debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/JdbcChangeEventSink.java

```diff
@@ -165,13 +155,25 @@ private void validate(JdbcSinkRecord record) {
         }
     }
 
-    private Buffer resolveBuffer(Map<CollectionId, Buffer> bufferMap, CollectionId collectionId, JdbcSinkRecord record) {
+    private List<JdbcSinkRecord> getRecordsToFlush(Map<CollectionId, Buffer> bufferMap, CollectionId collectionId, JdbcSinkRecord record) {
+        Stopwatch stopwatch = Stopwatch.reusable();
+        stopwatch.start();
+
+        Buffer buffer;
```

Comment:
```suggestion
        final Buffer buffer;
```
It avoids the use case where `buffer` is mistakenly left unassigned.